modules/core/src/main/scala/lucuma/odb/api/repo/TopLevelRepo.scala

```diff
@@ -195,20 +195,19 @@ abstract class TopLevelRepoBase[F[_]: Monad, I: Gid, T: TopLevelModel[I, *]: Eq]
 
   def constructAndPublish[U <: T](
     cons: Tables => ValidatedInput[State[Tables, U]]
-  ): F[U] = {
-    val fu = EitherT(
+  ): F[ValidatedInput[U]] = {
```

Comment:
So instead of throwing errors here we pass them up so they can be written in the query results.